__tests__/api/getContests.js

```diff
@@ -0,0 +1,14 @@
+import api from '../../src/api';
+import judges from '../../src/judges';
+jest.mock('../../src/judges');
+
+// I need more than one judge working to make better tests.
+test('concatenates correctly', () => {
+  judges.codeforces.getContests.mockImplementation(() => Promise.resolve([1, 2, 3]));
```

Comment:
Instead of using Contest, I'll use a revoked proxy. That way, `getContests` can't access any field of any of the contests (and I think it shouldn't), and if it tries we'll know.